### PR TITLE
CommonMark writer: do not throw away attributes when Ext_attributes is enabled

### PR DESCRIPTION
**Pull request by jtojnar**
`﻿Ext_attributes` covers at least the following:

- `Ext_fenced_code_attributes`
- `Ext_header_attributes`
- `Ext_inline_code_attributes`
- `Ext_link_attributes`

### CODE REVIEW

File: src/Text/Pandoc/Writers/Markdown.hs [hunk 1]
```diff
@@ -286,7 +286,7 @@ classOrAttrsToMarkdown attrs = attrsToMarkdown attrs
 
 linkAttributes :: WriterOptions -> Attr -> Doc Text
 linkAttributes opts attr =
-  if isEnabled Ext_link_attributes opts && attr /= nullAttr
```
Comment (jtojnar) [inline src/Text/Pandoc/Writers/Markdown.hs:289]: I wonder if we could just reuse the `linkAttributes` from `Inline`.

File: src/Text/Pandoc/Writers/Markdown.hs [hunk 2]
```diff
@@ -322,7 +322,7 @@ blockToMarkdown' opts (Div attrs ils) = do
     case () of
          _ | isEnabled Ext_fenced_divs opts &&
              attrs /= nullAttr ->
-                nowrap (literal ":::" <+> classOrAttrsToMarkdown attrs) $$
+                nowrap (literal ":::" <+> (if variant == Commonmark then attrsToMarkdown else classOrAttrsToMarkdown) attrs) $$
```
Comment (jgm) [inline src/Text/Pandoc/Writers/Markdown.hs:325]: Reminder to keep lines <= 80 chars.

Comment (jtojnar) [reply src/Text/Pandoc/Writers/Markdown.hs]: Sorry, fixed.

**Comment by jgm** [comment]
Looks good, thanks!

**Pull request merged**